lib/components/narrative/default/default-itinerary.js

```diff
@@ -194,53 +203,86 @@ class DefaultItinerary extends NarrativeItinerary {
-              })
-            }
+            {ITINERARY_ATTRIBUTES.sort((a, b) => {
+              const aSelected = this._isSortingOnAttribute(a)
+              const bSelected = this._isSortingOnAttribute(b)
+              if (aSelected) return -1
+              if (bSelected) return 1
+              else return a.order - b.order
+            }).map((attribute) => {
+              const isSelected = this._isSortingOnAttribute(attribute)
+              const options = attribute.id === 'arrivalTime' ? timeOptions : {}
+              if (isSelected) {
+                options.isSelected = true
+                options.selection = this.props.sort.type
+              }
+              options.LegIcon = LegIcon
+              options.configCosts = configCosts
+              options.currency = currency
+              return (
+                <li
+                  className={`${attribute.id}${isSelected ? ' main' : ''}`}
+                  key={attribute.id}
+                >
+                  {attribute.render(itinerary, options)}
+                </li>
+              )
+            })}
           </ul>
           <ItinerarySummary itinerary={itinerary} LegIcon={LegIcon} />
+          {itineraryHasAccessibilityScores(itinerary) && (
+            <AccessibilityRating
+              gradationMap={accessibilityScoreGradationMap}
+              large
+              score={getAccessibilityScoreForItinerary(itinerary)}
+            />
+          )}
           <FieldTripGroupSize itinerary={itinerary} />
-          {(active && !expanded) &&
+          {active && !expanded && (
             <DetailsHint>
               <FormattedMessage id='components.DefaultItinerary.clickDetails' />
             </DetailsHint>
-          }
+          )}
         </button>
-        {(active && expanded) &&
+        {active && expanded && (
           <>
             {showRealtimeAnnotation && <SimpleRealtimeAnnotation />}
-            <ItineraryBody itinerary={itinerary} LegIcon={LegIcon} setActiveLeg={setActiveLeg} timeOptions={timeOptions} />
+            <ItineraryBody
+              accessibilityScoreGradationMap={accessibilityScoreGradationMap}
+              itinerary={itinerary}
+              LegIcon={LegIcon}
+              setActiveLeg={setActiveLeg}
+              timeOptions={timeOptions}
+            />
           </>
-        }
+        )}
       </div>
     )
   }
 }
 
 const mapStateToProps = (state, ownProps) => {
+  const {intl} = ownProps
+  const gradationMap = state.otp.config.accessibilityScore?.gradationMap
+
+  // Generate icons based on fa icon keys in config
+  // Override text fields if translation set
+  Object.keys(gradationMap).forEach(key => {
+    const {icon} = gradationMap[key]
+    if (icon && typeof icon === 'string') {
+      gradationMap[key].icon = <Icon type={icon} />
+    }
+
+    const localizationId = `config.acessibilityScore.gradationMap.${key}`
+    const localizedText = intl.formatMessage({id: localizationId})
```

Comment:
Add a comment to remind that message ids must be dynamically generated here because they are found in config (and not in the standard language files).

Reply:
Comment added in 1aefcd05!